Gems/AtomLyIntegration/EditorModeFeedback/Assets/Shaders/EditorModeMask.material

```diff
@@ -1,6 +1,6 @@
 {
     "description": "",
     "parentMaterial": "",
-    "materialType": "Shaders/PostProcessing/EditorModeMask.materialtype",
+    "materialType": "Shaders/EditorModeMask.materialtype",
     "materialTypeVersion": 1
 }
```

Comment:
uber nit: Might as well add a eol character (unless this file is created automatically?)

Reply:
Fixed 👍